.github/workflows/builds.yml

```
@@ -263,34 +262,30 @@ jobs:
+        uses: ./.github/actions/dependencies
+        with:
+          compiler: ${{ matrix.cxx }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Restore ccache
         uses: actions/cache/restore@v4
         id: ccache-restore
         with:
           path: ${{ env.CCACHE_DIR }}
-          key: ccache-${{ runner.os  }}-${{ github.job }}-${{ env.CCACHE_KEY_SUFFIX }}-${{ github.sha }}
+          key: ccache-${{ runner.os  }}-${{ github.job }}-${{ env.CCACHE_KEY_SUFFIX }}-${{ matrix.image }}-${{ matrix.std }}-${{ matrix.compiler }}-${{ github.sha }}
```

actionlint (1.7.4)

property "compiler" is not defined in object type {cxx: string; image: string; std: number}

(expression)